alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/GlobalConfigExistsValidator.java

```diff
@@ -70,11 +70,22 @@ public GlobalConfigExistsValidator(ConfigurationAccessor configurationAccessor,
             return Optional.empty();
         }
 
-        String descriptorDisplayName = optionalDescriptorKey.map(DescriptorKey::getDisplayName).orElse(descriptorName);
+        String descriptorDisplayName = optionalDescriptorKey
+                                           .map(DescriptorKey::getDisplayName)
+                                           .orElse(descriptorName);
         try {
             List<ConfigurationModel> configurations = configurationAccessor.getConfigurationsByDescriptorNameAndContext(descriptorName, ConfigContextEnum.GLOBAL);
             if (configurations.isEmpty()) {
                 return Optional.of(String.format(GLOBAL_CONFIG_MISSING, descriptorDisplayName));
+            } else {
+                boolean configurationsAreEmpty = configurations
```

Comment:
Moving this stream to line 77.5 will allow you to have only the original if statement as streaming over an empty list will hardly have an impact to begin with.